### PR TITLE
API key update

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the user authentication documentation to address some recent changes.

- [Tenant Management API Key](https://deploy-preview-1666--docs-spectrocloud.netlify.app/tenant-settings/api-key-management)

- [User Authentication](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/)
- [Authorization Token](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/authorization-token)
- [UI Auth](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/ui-autentication)
- [API Key Section](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/api-key)
  - [Create API Key](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/api-key/create-api-key)
  - [Edit API Key](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/api-key/modify-api-key)
  - [Revoke API Key](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/api-key/revoke-api-key)
  - [Delete API Key](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/api-key/delete-api-key)

## Review Changes

💻 [Preview URL](https://deploy-preview-1666--docs-spectrocloud.netlify.app/user-management/authentication/)

🎫 [PEM-3373](https://spectrocloud.atlassian.net/browse/PEM-3373)
🎫 [PEM-3441](https://spectrocloud.atlassian.net/browse/PEM-3441)

### CODE REVIEW

File: docs/docs-content/user-management/authentication/api-key/delete-api-key.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+4. Click on **Delete**.
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/delete-api-key.md:59]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/api-key/delete-api-key.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+3. Verify your API key is not listed in the table.
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/delete-api-key.md:97]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+| **Attribute** | **Description** | **Edit** |
+| --- | --- | --- |
+| **API Key Name** | The user assigned name of the API key. | ✅ |
+| **Description** | An optional description about the key. | ✅ |
+| **Expiration Date** | The expiration date of the key. | ✅ |
+| **Value** | The API key value. | ❌ |
+
+
+Tenant administrators have the ability to edit an API key on behalf of any user within the tenant. Select the tenant tab below to learn more about editing an API key as a tenant admin.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:21]:
```suggestion
Tenant administrators can edit an API key on behalf of any user within the tenant. Select the Tenant tab below to learn more about editing an API key as a tenant admin.
```

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+:::
+
+## Prerequisites
+
+<Tabs groupId="scope">
+<TabItem label="User" value="user">
+
+  * You must have a Palette account, and you must be logged in.
+
+  * You must have an API key created. Refer to the [Create API Key](create-api-key.md) section for more information.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:36]: The first sentence in line 36 is ambiguous. Maybe: "An existing API key."

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+  * You must have a Palette account, and you must be logged in.
+
+  * An API key must be created for the user you want to edit the API key for. Refer to the [Create API Key](create-api-key.md) section for more information.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:43]: Maybe: "An existing API key."

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+5. Make the necessary changes to the API key attributes and click **Save**.
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:75]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Navigate to the **User Menu**, and select **My API Keys**.
+
+
+3. Identify your API key in the table.
+
+
+4. Verify the changes you made to the API key attributes are reflected in the table. Such as the API key name, description, and expiration date.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:109]:
```suggestion
4. Verify the changes you made to the API key attributes are reflected in the table, including API key name, description, and expiration date.
```

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:118]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+3. Navigate to the left **Main Menu** and select **Tenant Settings**.
+
+
+4. From the **Tenant Settings Menu**, select **API Keys**.
+
+
+5. Identify the altered API key in the table.
+
+
+6. Verify the changes you made to the API key attributes are reflected in the table. Such as the API key name, description, and expiration date.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:130]:
```suggestion
6. Verify the changes you made to the API key attributes are reflected in the table, including the API key name, description, and expiration date.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 10]
```diff
@@ -0,0 +1,231 @@
...
+sidebar_label: "Revoke API Key"
+title: "Revoke API Key"
+description: "Revoke an existing API Key in Palette."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["user-management", "authentication", "api-key"]
+---
+
+You can revoke an API key from Palette. A tenant admin can also revoke an API key created by another user within the tenant. A revoked API key cannot be used to authenticate API requests and requires a re-activation to be used again.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:11]: In the first sentence, is the key revoked from Palette or a Palette user?

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 11]
```diff
@@ -0,0 +1,231 @@
...
+
+
+Tenant administrators have the ability to revoke an API key in the tenant. Select the tenant tab below to learn more about revoking an API key as a tenant admin.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:14]:
```suggestion
Tenant administrators can revoke an API key in the tenant. Select the Tenant tab below to learn more about revoking an API key as a tenant admin.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 12]
```diff
@@ -0,0 +1,231 @@
...
+
+### Prerequisites
+
+<Tabs groupId="scope">
+<TabItem label="User" value="user">
+
+
+  * You must have a Palette account, and you must be logged in.
+
+  * You must have an API key created. Refer to the [Create API Key](create-api-key.md) section for more information.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:38]: The first sentence in line 38 is unclear. Maybe "An existing API key."

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 13]
```diff
@@ -0,0 +1,231 @@
...
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+  * You must have a Palette account, and you must be logged in.
+
+  * An API key must be created for the user you want to edit the API key for. Refer to the [Create API Key](create-api-key.md) section for more information.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:45]: Maybe: "An existing API key." (line 45)

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 14]
```diff
@@ -0,0 +1,231 @@
...
+
+
+4. Click on **Revoke**.
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:73]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 15]
```diff
@@ -0,0 +1,231 @@
...
+4. Verify the **Active** column in the table is set to **Inactive** for your API key.
+
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:115]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 16]
```diff
@@ -0,0 +1,231 @@
...
+
+### Prerequisites
+
+
+<Tabs groupId="scope">
+<TabItem label="User" value="user">
+
+  * You must have a Palette account, and you must be logged in.
+
+  * You must have an API key created. Refer to the [Create API Key](create-api-key.md) section for more information.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:143]: Maybe: "An existing API key." ?

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 17]
```diff
@@ -0,0 +1,231 @@
...
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+  * You must have a Palette account, and you must be logged in.
+
+  * An API key must be created for the user you want to edit the API key for. Refer to the [Create API Key](create-api-key.md) section for more information.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:150]: Maybe: "An existing API key." ?

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 18]
```diff
@@ -0,0 +1,231 @@
...
+4. Click on **Re-activate**.
+
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:180]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md [hunk 19]
```diff
@@ -0,0 +1,231 @@
...
+
+4. Verify the **Active** column in the table is set to **Active** for your API key.
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:219]:
```suggestion
2. Switch to the **Tenant Admin** scope.
```

File: docs/docs-content/user-management/authentication/authorization-token.md [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+sidebar_label: "Authorization Token"
+title: "Authorization Token"
+description: "Learn about Authorization Token authentication method in Palette."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["user-management", "authentication"]
+---
+
+Palette exposes an API endpoint to generate an authorization token for a user. The authorization token is valid for 15 minutes. You can refresh the token using the refresh token API. The authorization token is valid for 15 minutes. You can refresh the token using the refresh token API endpoint.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/authorization-token.md:11]:
```suggestion
Palette exposes an API endpoint to generate an authorization token for a user. The authorization token is valid for 15 minutes. You can refresh the token using the refresh token API endpoint.
```

File: docs/docs-content/user-management/authentication/authorization-token.md [hunk 21]
```diff
@@ -0,0 +1,77 @@
...
+
+API requests using the authroization token must use the HTTP header `Authorization` with token as the value.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/authorization-token.md:13]:
```suggestion
API requests using the authorization token must use the HTTP header `Authorization` with a token as the value.
```

File: docs/docs-content/user-management/authentication/authorization-token.md [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+```
+
+```bash
+curl --location --request GET 'https://api.spectrocloud.com/v1/projects/alert' \
+--header 'Authorization: $TOKEN' \
+--header 'Content-Type: application/json' 
+```
+
+
+To refresh the authorization token, use the `v1/auth/refresh` endpoint with a GET request to refresh the authorization token.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/authorization-token.md:26]:
```suggestion
To refresh the authorization token, use the `v1/auth/refresh` endpoint with a GET request.
```

File: docs/docs-content/user-management/authentication/authorization-token.md [hunk 23]
```diff
@@ -0,0 +1,77 @@
...
+
+
+## Aquire Authorization Token
+
+To acquire an authorization token, use the `v1/auth` endpoint with a POST request to generate the authorization token. Provide the email ID, organization name, and password in the request body. For example:
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/authorization-token.md:31]:
```suggestion
To acquire an authorization token, use the `v1/auth` endpoint with a POST request to generate the token. Provide the email ID, organization name, and password in the request body. For example:
```

File: docs/docs-content/user-management/authentication/authorization-token.md [hunk 24]
```diff
@@ -0,0 +1,77 @@
...
+    "Authorization": "eyJhbGc.........."
+}
+```
+
+Use the authorization token in the `Authorization` header to make API requests.
+
+
+## Refresh Authorization Token
+
+To refresh the authorization token, use the `v1/auth/refresh` endpoint with a GET request to refresh the authorization token. Provide the refresh token as a path parameter by replacing `:token` in the URL with the refresh token.
```
Comment (ritawatson) [inline docs/docs-content/user-management/authentication/authorization-token.md:55]:
```suggestion
To refresh the authorization token, use the `v1/auth/refresh` endpoint with a GET request. Provide the refresh token as a path parameter by replacing `:token` in the URL with the refresh token. 
```

**Review by ritawatson — APPROVED** [review]
Approving with suggestions. This was a heavy lift and a job well done.

File: docs/docs-content/user-management/authentication/api-key/create-api-key.md [hunk 25]
```diff
@@ -0,0 +1,153 @@
...
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Switch to the **Tenant Admin** scope
+
+
+3. Navigate to the left **Main Menu** and select **Tenant Settings**.
+
+ß
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply docs/docs-content/user-management/authentication/api-key/create-api-key.md:77]: Nice catch

File: docs/docs-content/user-management/authentication/api-key/delete-api-key.md [hunk 26]
```diff
@@ -0,0 +1,108 @@
...
+  * You must have an API key created. Refer to the [Create API Key](create-api-key.md) section for more information.
+
+</TabItem>
+<TabItem label="Tenant" value="tenant">
+
+  * You must have a Palette account, and you must be logged in.
+
+  * Tenant administrator access.
+
+  * An API key must be created for the user you want to delete the API key for. Refer to the [Create API Key](create-api-key.md) section for more information.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/delete-api-key.md:32]:
```suggestion
  * An existing API key must be available.
```

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md (hunk 4 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:36]:
```suggestion
  * An existing API key.
```

File: docs/docs-content/user-management/authentication/api-key/modify-api-key.md (hunk 5 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/modify-api-key.md:43]:
```suggestion
  * An existing API key.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md (hunk 10 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:11]:
```suggestion
You can revoke a user's API key from Palette. A tenant admin can also revoke an API key created by another user within the tenant. A revoked API key cannot be used to authenticate API requests and requires a re-activation to be used again.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md (hunk 12 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:38]:
```suggestion
  * An existing API key.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md (hunk 13 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:45]:
```suggestion
  * An existing API key.
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md (hunk 16 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:143]:
```suggestion
  * An existing API key..
```

File: docs/docs-content/user-management/authentication/api-key/revoke-api-key.md (hunk 17 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/api-key/revoke-api-key.md:150]:
```suggestion
  * An existing API key.
```

File: docs/docs-content/user-management/authentication/authorization-token.md (hunk 24 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/user-management/authentication/authorization-token.md]:
```suggestion
To refresh the authorization token, use the `v1/auth/refresh` endpoint with a GET request. Provide the refresh token as a path parameter by replacing `:token` in the URL with the refresh token. 
```

**Pull request merged**